core/tests/wdio.conf.js

```diff
@@ -15,9 +15,10 @@ var chromeVersion = (args[0] == 'DEBUG=true') ? args[6] : args[5];
 var chromedriverPath =
 './node_modules/webdriver-manager/selenium/chromedriver_' + chromeVersion;
 
-// To record videos of the failed test suites locally,
-// update the value of LOCAL_VIDEO_RECORDING_IS_ENABLED to 1.
-var LOCAL_VIDEO_RECORDING_IS_ENABLED = 0;
+var spw = null;
+var vidPath = '';
```

Comment:
Use videoPath, don't use abbreviations. Also, add a descriptive comment above this line explaining what this means and whether it's an absolute or relative path. If 'relative' specify what it's relative to.

Reply:
Done.